android-ndk-sys/src/lib.rs

```diff
@@ -20,14 +20,14 @@ pub mod native_app_glue;
 compile_error!("android-ndk-sys only supports compiling for Android");
 
 #[cfg(any(
-    all(any(target_os = "android", test), target_arch = "arm"),
+    all(
+        any(target_os = "android", test),
+        any(target_arch = "arm", target_arch = "armv7")
```

Comment:
You can see the actual compiler specs here:
https://github.com/rust-lang/rust/blob/master/src/librustc_target/spec/arm_linux_androideabi.rs#L15
https://github.com/rust-lang/rust/blob/master/src/librustc_target/spec/armv7_linux_androideabi.rs#L23
The arch for both is "arm" (nothing of that "armv7"). Right?